pcsx2-qt/MainWindow.ui

```diff
@@ -268,6 +268,15 @@
     <string>Start &amp;File...</string>
    </property>
   </action>
+  <action name="actionToolbarStartFile">
+   <property name="icon">
+    <iconset theme="file-line">
+     <normaloff>.</normaloff>.</iconset>
+   </property>
+   <property name="text">
+    <string comment="In Toolbar">Start &amp;File...</string>
```

Comment:
Please expand "In Toolbar" strings with i18n information so translators know that these strings should be abbreviated or shortened as much as possible.

Reply:
Nope. But you can do that on crowdin if you want additional context.